foodnet/membership/models.py

```diff
@@ -63,39 +67,43 @@ def __str__(self):
 
 class Member(models.Model):
```

Comment:
As far as I can tell the fields on `Member` could easily be on the `UserProfile` model since it's a one-to-many relation. Is the idea that many users can share the same member?